.github/CONTRIBUTING.md

```
@@ -153,4 +149,33 @@ For setup instructions, refer to [README.md](../infra/git/hooks/README.md).
 - If your change affects usage, update `README.md` or any relevant documentation.
 - Ensure all example commands and descriptions are up to date.
 
-
+# Code Style
```

markdownlint-cli2 (0.17.2)

Multiple top-level headings in the same document
null

(MD025, single-title, single-h1)